src/NuGet.Core/NuGet.Build.Tasks/Strings.resx

```diff
@@ -117,6 +117,12 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="ImportsFallbackWarning" xml:space="preserve">
+    <value>Project '{0}' was restored using '{1}' instead of the project target framework '{2}'. This project may not be fully compatible with your project.</value>
```

Comment:
nit: not too clear about the warning message for couple of reasons:
1. This is raised as part of build, not restore so may be more appropriate to use `resolve` instead of `restore`.
2. Host project should also be mentioned otherwise it will be pain to relate it in command line.

May be something like 
`
Project {0} P2P reference {1} resolved using {2} instead of the project target framework {3}. This..........
{0} - Host Project
{1} - P2P reference
{2} - fallback tfm
{3} - original project tfm
`

Reply:
Or

```
Project '{0}' is referenced using target framework '{1}' instead of this project's target framework '{2}'. This project may not be fully compatible with your project.
```

?